codex/uberdev-codex/lib/command-workspace.py

```
@@ -51,12 +52,70 @@
+    if effective_uid() is None or not secure_dir_fd_available():
+        fail("unsupported_platform")
+    return "descriptor_relative"
+
+
+def state_directory_name() -> str:
+    uid = effective_uid()
+    if uid is not None:
+        return f".agent-state-{uid}"
+    if native_windows():
+        return ".agent-state-0"
+    fail("unsupported_platform")
```

Ruff (0.15.21)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)